resources/options.html

```diff
@@ -9,6 +9,7 @@
     <script src="jquery.min.js"></script>
     <script src="materialize.min.js"></script>
     <script src="options.js"></script>
+    <!--TODO: add the do not watch options -->
```

Comment:
What typescript file are you talking about? this is just a memo that UI should be made for these new options.